src/features/MentorPage/components/MentorList/MentorCard/List/index.tsx

```diff
@@ -22,6 +24,12 @@ export const ListCard: React.FC<Props> = ({
   mentor,
 }) => {
   const { isMobile } = useGetLayoutMode();
+  const currentUserId = useAppSelector(selectUserId);
+
+  const today = new Date();
+  const threeMonthsAgo = new Date();
+  threeMonthsAgo.setMonth(today.getMonth() - 3);
+  const timeThreeMonthsAgo = threeMonthsAgo.getTime();
```

Comment:
Oops missed this last time reviewing.
I don't like to see mutating of variables => 

```javascript
const foo = {}
foo['bar'] = 'baz'
```
Reason is that sometimes weird bugs happen and it can be difficult to figure out why.
And in this case we do it with

`threeMonthsAgo.setMonth`

I would change this just by 

`const threeMonthsAgo = new Date().setMonth(new Date().getMonth()-3)`

But since this is your first PR and you are going to continue working on this topic anyway it can be done in the next PR

Reply:
And mutating variables is just something i am allergic to. x) it is ok when we do it inside a function etc, but still I prefer to avoid it.

Reply:
Ok, thank you!